pkg/lang/javascript/aws_runtime/_emitter.ts

```diff
@@ -22,11 +22,7 @@ export class Emitter extends events.EventEmitter {
     private client: SNSClient
     private s3: S3Client
 
-    constructor(
-        private path: string,
-        private name: string,
-        private id: string
-    ) {
+    constructor(private path: string, private name: string, private id: string) {
```

Comment:
I think you need to update your prettier - I believe Jordan ran into this failing the check in GH action earlier.